claudio-plugin/skills/jira-utilities/scripts/update_issue.sh

```
@@ -78,12 +98,14 @@ if [[ -n "$PRIORITY" ]]; then
         '. + {additionalAttributes: {priority: {name: $v}}}')
 fi
 
-# ---- Write temp file and edit via acli ----
+# ---- Include key in JSON (acli requires issues array, not key field) ----
+JSON=$(echo "$JSON" | jq --arg k "$KEY" '. + {issues: [$k]}')
+
 TMPFILE=$(make_tmp_json "$JSON")
 trap "rm -f $TMPFILE" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)